website/content/docs/concepts/request-limiter.mdx

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+page_title: 'Request Limiter'
+description: >-
+  Vault provides an adaptive concurrency limiter to protect the Vault server
+  from overload.
+---
+
+# Request Limiter
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/beta.mdx'
+
+This document contains conceptual information about the **Request Limiter** and
+its user-facing effects.
+
+## Preventing overload
+
+The Request Limiter aims to prevent overload by proactively detecting latency
```

Comment:
```suggestion
The Request Limiter aims to prevent overload by proactively detecting latency increases from a baseline
```

Reply:
I addressed this in https://github.com/hashicorp/vault/pull/25557/commits/7a38c1ff7ee0448f14b115478f967947a06e704e and used "deviation" instead of "increases". Let me know what you think!